ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/BlockResult.java

```diff
@@ -89,6 +102,7 @@
     this.timestamp = Quantity.create(header.getTimestamp());
     this.ommers = ommers;
     this.transactions = transactions;
+    this.coinbase = includeCoinbase ? header.getCoinbase().toString() : null;
```

Comment:
I think it's cleaner to just always include `coinbase` instead of adding a flag to switch on this.  Shouldn't hurt anything if there's extra data returned that users aren't expecting.  Kind of odd that it's not a standard field anyway.

Reply:
Not only is it not in the JSON-RPC Spec, what reteseth wants is an `author` field which has the same meaning test wise as `coinbase`.  If we expose it for all I would still want a flag and we expose `coinbase` not `author`.